### PR TITLE
Feature/add new arr import scripts

### PR DESCRIPTION
**Pull request by BuSHari**
A new and improved Arr scripts for Manual Import scripts

[X] I agree that by opening a pull requests I am handing over copyright ownership of my work contained in that pull request to the FileFlows project and the project owner. My contribution will become licensed under the same license as the overall project.

### CODE REVIEW

File: Scripts/Flow/Applications/Radarr/Radarr - Trigger Manual Import.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/**
+ * @name Radarr - Trigger Manual Import 2.0
+ * @uid cd8926f3-a9ab-4fa7-a36e-cceb8ad58987
+ * @description Trigger Radarr to manually import the media, run after last file in folder moved.
+ * @author iBuSH
+ * @revision 1
+ * @param {string} URL Radarr root URL and port (e.g. http://radarr:7878)
+ * @param {string} ApiKey Radarr API Key
+ * @param {string} ImportPath The output path for import triggering (default Working File)
+ * @param {bool} UseUnmappedPath Whether to Unmap the path to the original FileFlows Server path when using nodes in different platforms (e.g. Docker and Windows)
```
Comment (revenz) [inline Scripts/Flow/Applications/Radarr/Radarr - Trigger Manual Import.js:12]: these comments are wrong, it only has 2 parameters, yet you have comments for more

File: Scripts/Flow/Applications/Sonarr/Sonarr - Trigger Manual Import.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * @author iBuSH
+ * @revision 1
+ * @param {string} URL Sonarr root URL and port (e.g. http://sonarr:8989)
+ * @param {string} ApiKey Sonarr API Key
+ * @param {string} ImportPath The output path for import triggering (default Working File)
+ * @param {bool} UseUnmappedPath Whether to Unmap the path to the original FileFlows Server path when using nodes in different platforms (e.g. Docker and Windows)
+ * @param {bool} MoveMode Import mode 'copy' or 'move' (default copy)
+ * @output Command sent
+ */
+function Script(URL, ApiKey) {
```
Comment (revenz) [inline Scripts/Flow/Applications/Sonarr/Sonarr - Trigger Manual Import.js:16]: these comments are wrong too

**Review by revenz — CHANGES_REQUESTED** [review]

File: Scripts/Flow/Applications/Radarr/Radarr - Trigger Manual Import.js (hunk 1 above)
Comment (BuSHari) [reply Scripts/Flow/Applications/Radarr/Radarr - Trigger Manual Import.js:12]: Fixed

File: Scripts/Flow/Applications/Sonarr/Sonarr - Trigger Manual Import.js (hunk 2 above)
Comment (BuSHari) [reply Scripts/Flow/Applications/Sonarr/Sonarr - Trigger Manual Import.js]: Fixed

**Pull request merged**